tutorials/plot_ecog.py

```diff
@@ -0,0 +1,47 @@
+"""
+======================
+Working with ECoG data
+======================
+
+MNE supports working with more than just MEG and EEG data. Here we show some
+of the functions that can be used to facilitate working with
+electrocorticography (ECoG) data.
+"""
+# Author: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+from scipy.io import loadmat
+from mayavi import mlab
+
+import mne
+from mne.viz import plot_trans
+
+print(__doc__)
+
+###############################################################################
+# Let's load some ECoG electrode locations and names, and turn them into
+# a :class:`mne.DigMontage` class.
+
+mat = loadmat(mne.datasets.misc.data_path() + '/ecog/sample_ecog.mat')
```

Comment:
It's some electrodes from a real subject that I inaccurately morphed to `sample` for demonstration purposes, yes